### PR TITLE
Added Aiven for Apache Kafka Connect home page, and first content

### LINKED ISSUE #361: Migrate Kafka Connect connectors article
# What should we add to our content collections?

Migrating Kafka Connect Connectors article https://help.aiven.io/en/articles/1231452-kafka-connect-connectors

# Is this something you would work on yourself?

[X] I plan to open a pull request for this

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters

Added Aiven for Apache Kafka Connect home page, getting-started.
The list of connectors as concept and enabling kafka connect as howto 

Fixes #361

### CODE REVIEW

**Comment by ftisiot** [comment]
Fixes #361

File: docs/products/kafka/kafka-connect/index.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Aiven for Apache Kafka Connect is a fully managed **distributed Apache Kafka integration component**, deployable in the cloud of your choice. Apache Kafka Connect lets you integrate your existing data sources and sink with Apache Kafka.
+
+With a Apache Kafka Connect connector, you can source data from an existing technology into a topic or sink data from a topic to a target technology by just defining the endpoints.
+
+
+Why Apache Kafka Connect?
+-------------------------
+
+Apache Kafka represents the best in class data streaming solution. Apache Kafka Connect enables to integrate Apache Kafka with the rest of your data architecture by setting a configuration file defining the source and the target of your data.
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/index.rst:15]: To make it easier to read I'd replace "by setting a configuration file defining the source and the target of your data." with "with a help of configuration file which defines the source and the target of your data"

File: docs/products/kafka/kafka-connect/getting-started.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
+Getting started
+===============
+
+Aiven services are managed in the Aiven `web console <https://console.aiven.io/>`__ . When you first log in to the console with your email address and password, you will see the **Services** view, which shows you all the services in the currently selected project.
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/getting-started.rst:4]: "in to" should be "into"

Comment (anelook) [inline docs/products/kafka/kafka-connect/getting-started.rst:4]: What if we make it more relatable for the reader and replace "Aiven services are managed in the Aiven " with "You can manage the Aiven services in our web console"

File: docs/products/kafka/kafka-connect/getting-started.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+.. Warning::
+
+    Aiven for Apache Kafka Connect services can be created only on top of existing Aiven for Apache Kafka running services. 
+    If your project doesn't contain any Aiven for Apache Kafka services, create one before attempting to create an Aiven for Apache Kafka Connect service.
+
+Create a new Aiven for Apache Kafka Connect dedicated service:
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/getting-started.rst:11]: I wonder if adding a title and bringing this into a subsection will help to scan content quickly

Creating Aiven for Apache Kafka Connect
------------------------------------------

Follow these steps to create a new Aiven for Apache Kafka Connect dedicated service:

Comment (anelook) [reply docs/products/kafka/kafka-connect/getting-started.rst:11]: Or is it too much extra info?

File: docs/products/kafka/kafka-connect/getting-started.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    different providers and regions. The service summary on the
+    right side of the console shows you the pricing for your
+    selected options.
+
+6. Select a service plan. This defines how many servers and what kind of memory, CPU, and disk resources are allocated to your service.
+
+7. Click **Create Service** under the summary on the right side of the console. This brings you back to the **Services** view. Your new service is listed with a status indicator to show that it is being created.
+
+
+While your service is being built, you can visit the **Service overview** page and
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/getting-started.rst:33]: It sounds a bit confusing to me, what if we rephrase it to be more generic, something like this -
"You can see the service status in the **Service overview** page , wait till it changes from *REBUILDING* to *RUNNING* to be able to use its full functionality."

File: docs/products/kafka/kafka-connect/concepts.rst [hunk 5]
```diff
@@ -0,0 +1,6 @@
+Concepts
+========
+
+Explanations and supporting concepts to help you succeed with Aiven for Apache Kafka Connect.
+
+.. tableofcontents::
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/concepts.rst:6]: This will use proper anchors in the future? Now all links bring to the same top of the page...

File: docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst [hunk 6]
```diff
@@ -0,0 +1,104 @@
+List of available Apache Kafka Connect connectors
+=================================================
+
+The following Apache Kafka Connect connectors are currently available, and can
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:4]: Do we have to use "Apache Kafka Connect " in "Apache Kafka Connect connectors", when so many words are repeating, makes it difficult to read and process.

Comment (anelook) [inline docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:4]: I'd remove "are currently available", since it must be a given (we don't list those that are unavailable). then the sentence become way easier to read - 
"The following connectors can be used in any Aiven for Kafka services with Kafka Connect enabled."

File: docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+be used in any Aiven for Kafka services with Kafka Connect enabled. 
+
+
+Source connectors
+-----------------
+
+Source connectors enable the integration of data from an existing technology into an Apache Kafka topic. The following is the list of available connectors:
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:11]: "list of available connectors:" I'd change. " list of available source connectors:"

File: docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+* Schema Source
+
+* `Stream Reactor Cassandra <https://docs.lenses.io/connectors/source/cassandra.html>`__
+
+* `Stream Reactor MQTT <https://docs.lenses.io/connectors/source/mqtt.html>`__
+
+Sink connectors
+-----------------
+
+Sink connectors enable the integration of data from an existing Apache Kafka topic to a target technology. The following is the list of available connectors:
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:38]: "list of available connectors:" I'd change. " list of available sink connectors:"

File: docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+   :alt: Preview icon next to a MongoDB Apache Kafka Connect connector
+
+Some of the available connectors have the :badge:`preview,cls=badge-secondary badge-pill` tag next to the name. **Preview connectors do not come under our SLA**, consider this before using them for production purposes. 
+Bugs should be reported to the code owner directly.
+
+
+Requesting new connectors
+-------------------------
+
+Upon creating a support requests, Aiven will evaluate the requested connector and might add support for it.
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:90]: Should we suggest "You can request adding support of a new connector by creating a support ticket. Aiven will evaluate the requested connector and might add support for it."

Comment (anelook) [inline docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:90]: Are we referring us, Aiven, in third person? "We will then evaluate your request" sounds more friendly and less formal.

File: docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+Upon creating a support requests, Aiven will evaluate the requested connector and might add support for it. 
+
+Aiven evaluation process for new Apache Kafka Connect connectors checks:
+
+* license compatibility
+* technical evaluation
+* active repository maintenance
+
+The addition of new Apache Kafka connectors is done at the sole discretion of Aiven.
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:98]: This is such a formal phrasing. Hm, do we have to keep it so formal, or can we rephrase in friendlier words?

File: docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+
+.. Tip::
+
+    When requesting connectors that are not on the pre-approved list through a support ticket, specify the target Aiven for Apache Kafka service you'd like to have it installed to.
+
+If you know about new and interesting connectors you'd like Aiven to support, please open a support request about it to help us shaping the future roadmap.
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:104]: I'd add this line earlier as a starter of this block

File: docs/products/kafka/kafka-connect/howto/enable-connect.rst [hunk 12]
```diff
@@ -0,0 +1,17 @@
+Enable Kafka Connect on Aiven for Apache Kafka
+==============================================
+
+Kafka Connect can be enabled on the same nodes running Aiven for Apache Kafka with **business** or **premium** plans.
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/howto/enable-connect.rst:4]: Can it be enabled on different nodes? I wonder if we rephrase 

To enable  Kafka Connect for Aiven for Apache Kafka your services should be running on **business** or **premium** plans.

Do we loose some meaning?

**Review by anelook — COMMENTED** [review]
Yay! We have a starting point for Kafka Connect! 
I've added comments for some potential improvements and on the parts which I didn't fully understand.

File: docs/products/kafka/kafka-connect/index.rst [hunk 13]
```diff
@@ -0,0 +1,43 @@
+Aiven for Apache Kafka Connect
+==============================
+
+What is Aiven for Apache Kafka Connect?
+---------------------------------------
+
+Aiven for Apache Kafka Connect is a fully managed **distributed Apache Kafka integration component**, deployable in the cloud of your choice. Apache Kafka Connect lets you integrate your existing data sources and sink with Apache Kafka.
```
[the remark this answers is not in the document]
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/index.rst:7]: Thanks!

File: docs/products/kafka/kafka-connect/index.rst (hunk 1 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/index.rst:15]: fixed

File: docs/products/kafka/kafka-connect/getting-started.rst (hunk 2 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/getting-started.rst:4]: fixed

Comment (ftisiot) [reply docs/products/kafka/kafka-connect/getting-started.rst:4]: I took this piece from the main Aiven for Apache Kafka, not sure if we should change both

File: docs/products/kafka/kafka-connect/getting-started.rst (hunk 3 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/getting-started.rst:11]: I added as dedicated service

File: docs/products/kafka/kafka-connect/getting-started.rst (hunk 4 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/getting-started.rst:33]: fixed

File: docs/products/kafka/kafka-connect/concepts.rst (hunk 5 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/concepts.rst:6]: This is standard across all pages, not sure what we should do differently, maybe a topic for #devportal ?

File: docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst (hunk 6 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:4]: I know, but it's the used wording in this space, not sure how and if we should avoid it

Comment (ftisiot) [reply docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:4]: fixed

File: docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst (hunk 8 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:38]: fixed

File: docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst (hunk 9 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:90]: fixed

Comment (ftisiot) [reply docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:90]: not sure how we should refer to ourselves, maybe another topic for #devportal?

File: docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst (hunk 10 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:98]: I know, I took it from the existing content, not sure if and how we should rephrase it

File: docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst (hunk 11 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst:104]: moved

File: docs/products/kafka/kafka-connect/howto/enable-connect.rst (hunk 12 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/howto/enable-connect.rst:4]: Welll, we need to say that with this method we are running Kafka connect on the same nodes where Kafka is running, you could always create a dedicated cluster. I'll rephrase

**Review by anelook — APPROVED** [review]
Great! Thank you!
